src/components/Notifications.vue

```diff
@@ -1,7 +1,7 @@
 <script setup>
 import { useNotifications } from '@/composables/useNotifications';
 
-const duration = 4000;
+const duration = 4000000;
```

Comment:
Is this on purpose? We shouldn't change the duration

Reply:
Fixed, that was only for testing purposes